Packs/SafeBreach/Integrations/SafeBreach_v2/test_data/remediation_data_with_unvaild_string.json

```
@@ -0,0 +1,21 @@
+{
+  "remediationData": {
+    "Attack": {
+      "814": {
+        "SHA256": [
+          "109c702578b261d0eda01506625423f5a2b8cc107b0d8dfad84d39fb02bfa5cb"
+        ],
+        "Drop Path": [
+          "SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Winlogon\\SpecialAccounts\\UserList"
+        ]
+      }
+    }
+  },
+  "mitigationVendors": [
+    {
+      "id": "splunk",
+      "fileExport": false,
+      "searchQuery": "(file_hash IN (1d02e38505684c59f7e72b690c39409c8b7a62c2fffe097febf857d99814470e)"
```

Also, make sure about searchQuery parameter.
If it isn't a secret add it to .secrets-ignore file.
 Adding my suggestion:

```suggestion
      "searchQuery": "(file_hash IN (1111111111111111111111111111111111111111111111111111111111111111)"
```